docs/trex_getstarted.md

```diff
@@ -34,7 +35,14 @@ If you want to create an extension or work with the sample code, make sure you h
 
 You need Node.js and nmp to run the dashboard extension demos. Node.js is a JavaScript runtime. npm is a package manager for Node.js and is installed when you install Node.js.
 
+Requirements for using dashboard extensions in Tableau:
 
+* Tableau Desktop 2018.2 and later
+* Tableau Server 2018.2 and later
+* Tableau Online
+
+To run extensions on Tableau Server or Tableau Online, support for extensions must be enabled, and depending upon the extension and the data access it requires, the extension might need to be added to safe list. See
```

Comment:
@johnDance  I'm not sure how much we need to get into the safe list and admin controls here. Particularly, if  they are changing.  How is this: 

To run extensions on Tableau Server or Tableau Online, support for extensions must be enabled, and depending upon the extension and the data access it requires, the extension might need to be added to the safe list. To ensure security, administrators can control which extensions are allowed to run and can grant extensions access to summary or full data. Administrators can also block extensions from running. See...

Reply:
Sorry. I was just suggesting that we add "a" to the existing sentence. So rather than "added to safe list", it reads "added to a safe list". Perhaps it would be better "added to the site safe list". I agree that we don't need to get into other details here. Sorry for not being clear.

Reply:
ah. My fault. I didn't read your comment correctly!